alt_src/alt_src.py

```diff
@@ -2141,6 +2140,9 @@ def setup_logging(options):
     file_log_level = min(file_log_level, output_log_level)
     options.file_log_level = file_log_level
 
+    # remove the default handler of root logger
+    logger.handlers= []
```

Comment:
Formatting is slightly weird here.

```suggestion
    logger.handlers = []
```